Automattic/wp-calypso
# Marketplace Thank You: Footer product-type specific

Fixes #74525

## Proposed Changes

* Split the logic for footer next steps
* Add a logic for when to show each section, based on the presence of plugins or themes on the page

## Testing Instructions

* Go to the Thank you page with a selected plugin and check if the footer layout matches the image below. Route: `/marketplace/thank-you/:site?plugins=:plugin-slug`
* Go to the Thank you page with a selected theme and check if the footer layout matches the image below. Route: `/marketplace/thank-you/:site?themes=:theme-slug`
* Go to the Thank you page with a selected plugin and theme, and check if the footer layout matches the image below. Route: `/marketplace/thank-you/:site?plugins=:plugin-slug&themes=:theme-slug`

| Plugins  | Themes | Plugins and Themes |
| ------------- | ------------- | ------------- |
|<img width="976" alt="CleanShot 2023-03-21 at 17 12 36@2x" src="https://user-images.githubusercontent.com/5039531/226731672-fe546db1-e2b5-48d7-b09c-4ba77f18c985.png">|<img width="661" alt="CleanShot 2023-03-21 at 17 12 47@2x" src="https://user-images.githubusercontent.com/5039531/226731691-bcdc21f6-b870-4aed-b7d4-74787b42a400.png">|<img width="997" alt="CleanShot 2023-03-21 at 17 12 52@2x" src="https://user-images.githubusercontent.com/5039531/226731610-b54df86d-68c9-4b91-b5d9-de7c32b72a6c.png">|


## Pre-merge Checklist
